deis-cleanup.rb

```diff
@@ -30,11 +31,30 @@
   puts output
 end
 
+remove_containers = lambda do |container|
+  command = "docker rm #{container[:id]}"
+  command.prepend "echo #{container[:name]}: " if DRY_RUN
+  output, _status = Open3.capture2e(command)
+  puts output
+end
+
+output, status = Open3.capture2e("docker ps -a --format '{{.ID}};{{.Names}}' --filter 'exited=0'")
+abort output unless status.success?
+
+rows = CSV.parse(output, col_sep: ";")
+containers = rows.map { |row| %i(id name).zip(row).to_h }
+
 puts "deis-cleanup started."
 
+if CLEANUP_CONTAINERS
+  puts "removing containers"
+  containers.reject { |container| container[:name].match(EXCLUDE_REGEX) }.each(&remove_containers)
+end
```

Comment:
oh wait.. do we even need to fetch exited containers when `CLEANUP_CONTAINERS` is set to false?